pkg/flagbuilder/build_flags.go

```diff
@@ -29,8 +29,18 @@ import (
 	"github.com/golang/glog"
 )
 
-// BuildFlags builds flag arguments based on "flag" tags on the structure
+// BuildFlags returns a space seperated list arguments
```

Comment:
We should probably mark this deprecated (just with a comment saying to prefer calling BuildFlagsList)